crates/trade/src/lib.rs

```diff
@@ -66,11 +65,6 @@ pub struct TradeParams {
     /// A duration that defines how long the contract is meant to be valid.
     /// The coordinator calculates the maturity timestamp based on the current time and the expiry.
     pub expiry: Duration,
-
-    /// The public key of the oracle to be used
-    ///
-    /// The orderbook decides this when matching orders.
-    pub oracle_pk: XOnlyPublicKey,
```

Comment:
🙃 I would keep it as part of the trade params and add the oracle URL. I think the orderbook is set to be the authority for deciding this. Not a big deal though, we can also add / change it later.